pkg/oci/containerize_go.go

```diff
@@ -55,6 +54,10 @@ func (c goLayerBuilder) Build(cfg *buildConfig, p v1.Platform) (desc v1.Descript
 }
 
 func goBuild(cfg *buildConfig, p v1.Platform) (binPath string, err error) {
+	if cfg.tester != nil && cfg.tester.emulateSlowBuild {
```

Comment:
Embedding "if testing do X" in core logic is not idea, but in this case I feel it is warranted since the complexity of adding a mockable builder implementation is so much greater than a simple "if testing slowness, pause here"